src/openforms/js/components/admin/form_design/registrations/zgw/ZGWOptionsForm.js

```diff
@@ -0,0 +1,63 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import Field from 'components/admin/forms/Field';
+
+import ZGWFormFields from './ZGWOptionsFormFields';
+
+const ZGWOptionsForm = ({index, name, label, schema, formData, onChange}) => {
```

Comment:
Is the `index` prop provided anywhere? The form is called here:
https://github.com/open-formulieren/open-forms/blob/beb9c7c336868abbd43b487ffa22e1d99bf3b8f5/src/openforms/js/components/admin/form_design/RegistrationFields.js#L36-L55

But I don't see any `index` argument

Reply:
Yes, this have been added as a prop by me in this PR (line 43 `index={index}`), it's not present in master.